lib/outputmanager.js

```diff
@@ -606,12 +599,18 @@ OutputPlugin.prototype.writeThemeVariables = function(courseId, theme, themeVari
       });
     },
     function(seriesCallback) {
-      // Write the new LESS file
+      // Delete old/write new LESS file
+      if (modifiedProperties.length === 0) {
+        // No theme customisation, clear file
+        fs.removeSync(path.join(destinationFolder, Constants.Folders.Less, Constants.Folders.CustomStyleDirectory, Constants.Filenames.ThemeVariables));
+        return seriesCallback(null);
+      }
+
       fs.outputFile(destinationFile, modifiedProperties, 'utf8', function(err) {
         if (err) {
           logger.log('error', 'Theme customisations 4 of 4');
         }
-        seriesCallback(err);
+        seriesCallback(null);
```

Comment:
Did you mean to change from `err` to `null`? Looks like any error would now be ignored.